.github/workflows/koupleless_adapter_snapshot.yml

```
@@ -0,0 +1,79 @@
+name: Koupleless Adapter Snapshot
+on:
+  workflow_dispatch:
+
+jobs:
+  relase_for_jdk8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)